drivers/modem/gsm_ppp.c

```diff
@@ -302,7 +304,18 @@ static void set_ppp_carrier_on(struct gsm_modem *gsm)
 	}
 
 	api = (const struct ppp_api *)ppp_dev->api;
-	api->start(ppp_dev);
+	if (is_booting_phase) {
+		api->start(ppp_dev);
+	} else {
+		int status;
+		struct net_if *network_int;
+		network_int = net_if_get_default();
```

Comment:
Please place empty line after the variables have been declared.
Also this might return incorrect network interface if there are more than one in the system. Change this to
`iface = net_if_get_first_by_type(&NET_L2_GET_NAME(PPP));`

Reply:
addressed